provisioning/salt/roots/salt/pypi/config/pypi-syslog.logrotate.conf

```diff
@@ -1,10 +1,11 @@
 /var/log/cdn/{{ syslog_name }}/*.log {
-    daily
-    rotate 3
+    rotate 24
     missingok
     notifempty
     compress
     sharedscripts
+    dateext
+    dateformat -%Y-%m-%d-%s
```

Comment:
wish this could have a better timestamp, but working with what `logrotate` gives us.